content/en/docs/02/06_masters.md

```diff
@@ -4,3 +4,235 @@ weight: 26
 sectionnumber: 2.6
 ---
 
+In this lab we will learn how to replace a failed master node. As long as we do not lose the majority of out master nodes, we can simply replace the failed master node(s). If we lose our quorum, we need to restore the state of `etcd` from a previously created snapshot, which will not be covered in this lab.
+
+
+## Task {{% param sectionnumber %}}.1: Replacing a master node
+
+Since the master nodes are hosting the `etcd` it is not as straight forward as replacing a worker node backed by a MachineSet.
```

Comment:
```suggestion
Since the masters are hosting `etcd`, the key-value store that holds all the cluster's information and state, it is not as straightforward as replacing a worker node backed by a machine set.
```